### PR TITLE
Add state resetter

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

File: config/web.php [hunk 1]
```diff
@@ -16,6 +16,9 @@
             $params['yiisoft/session']['session']['options'],
             $params['yiisoft/session']['session']['handler'],
         ],
+        'reset' => function () {
+            $this->destroy();
```
Comment (samdark) [inline config/web.php]: Wouldn't that result in losing session data between requests?

Comment (rustamwin) [reply config/web.php]: $this->clear() should be here instead?

Comment (yiiliveext) [inline config/web.php]:
```suggestion
            $this->sessionId = null;
            $this->close();
```

**Review by samdark — APPROVED** [review]

**Comment by vjik** [comment]
Need changelog

**Pull request merged**